sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Release History
```

Comment:
Just wanted to clarify here, do we not need a hotfix branch for this release ?
https://github.com/Azure/azure-sdk/blob/master/docs/policies/repobranching.md#hotfix-branches

Reply:
I thought about it but because there haven't been changes in the FR code in between versions then it should be the same.
Azure Core dependency changed but it is always good to have the latest of latest

Reply:
I think that should work!